apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,68 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  // Demo saveConclusion function
+  const saveConclusion = () => {
+    // Demo getRegistryAnswer
+    const getRegistryAnswer = (id) => {
+      // How do we get the registry answer?
+      return "true";
+    };
+
+    // Demo getQuestionAndAnswers
+    const getQuestionAndAnswers = () => {
+      return this.stack.reduce((acc, question) => {
+        acc.push({
+          ...question,
+          // id: does `question` contain `id`? (Yes)
+          // text: does `question` contain `text`?
```

Comment:
We have the `text`, `description` and `longDescription`. I think `text` is enough.

Reply:
We have all questionAnswers from `getQuestionAnswers()` (what we made for feature/refresh) which is a list if question id's and its answers. I think thats good enough, and questionText will be a bit too much maybe. I don't know what happens when question id's change in JSON updates though... What do you think?

Reply:
If we have our own id's and not the one from FL, this is solved!